app/helpers/contentful_helper.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# Helper module for contentful
+# see also the contentful gem: https://github.com/contentful/contentful.rb
+module ContentfulHelper
+  def contentful
+    @client ||= Contentful::Client.new(
```

Comment:
Let's read the env vars into `contentful` config object in the initializer https://github.com/europeana/europeana-contribute/blob/master/config/initializers/env.rb